projects/igniteui-angular/src/lib/grids/filtering/grid-filtering-row.component.ts

```diff
@@ -367,7 +367,9 @@ export class IgxGridFilteringRowComponent implements AfterViewInit {
      * Commits the value of the input.
      */
     public commitInput() {
-        this.chipsArea.chipsList.filter(chip => chip.selected = false);
+        this.expressionsList.forEach(ex => ex.isSelected = false);
+        this.chipsArea.chipsList.forEach(chip => chip.selected = false);
```

Comment:
You shouldn't need to do this, since the `[selected]` of the `igx-chip` is bound to `isSelected`, which means that changing `isSelected` on the line above should update the chip selected input values.

Reply:
Yes, but clicking on a selected chip(to remove selected state and commit the filter value) doesn't work without second line.